tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -786,6 +798,52 @@ def _model_stl_thumb(self, filepath: Path, size: int) -> Image.Image:
 
         return im
 
+    def _pdf_thumb(self, filepath: Path, size: int) -> Image.Image:
```

Comment:
it would be nice to have a test for this. Rather than explaining what and how, I made the test, so feel free to cherry-pick/merge it into the branch. But it looks like the final png isnt really transparent 🤔 

https://github.com/TagStudioDev/TagStudio/commit/b1bae1f28aa4be6f54896cf0943dcea0d2b25bcd